src/index.js

```diff
@@ -8,15 +8,26 @@ import Root from './views/Root';
 export default class giVR extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { isFullScreen: false, fullScreenId: 0, images: [] };
+    this.state = {
+      isFullScreen: false,
+      fullScreenId: 0,
+      images: [{ thumbnail: 'empty', width: 1000, height: 600 }],
+    };
 
     // This binding is necessary to make `this` work in the callback
     this.handleClick = this.handleClick.bind(this);
   }
 
   componentDidMount() {
-    axios.post('http://localhost:3000/unsplash', { query: 'mock' }).then(({ data: images }) => {
-      this.setState({ images });
+    axios.post('http://localhost:3000/google', { query: 'mock' }).then(async ({ data: images }) => {
+      const data = await Promise.all(
+        images.map(async item => ({
+          ...item,
+          src: (await axios.get(`http://localhost:3000/image?url=${item.src}`)).data,
```

Comment:
not for this PR but we have to find a way to configure that "localhost" thing so that it can be used from different devices